app/org/sagebionetworks/bridge/models/studies/PasswordPolicy.java

```diff
@@ -0,0 +1,85 @@
+package org.sagebionetworks.bridge.models.studies;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The policies that govern password creation. New studies will be created with the default password 
+ * policy, which requires 8 characters, one numeric, one symbol and one upper- and lower-case 
+ * character. However, this policy can be adjusted for legacy studies which were created with much 
+ * more lax constraints.
+ */
+public final class PasswordPolicy {
```

Comment:
I still don't think we should expose this to researchers. A lot of companies are really bad at password policies. It would be better for us to just have a reasonable default password policy and create all studies going forward with that policy.